package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "editmode-react",
-  "version": "3.1.1",
+  "version": "3.1.2",
```

```suggestion
  "version": "3.1.1",
```

You don't need to bump the version here manually. `auto` will take care of it 👍